demisto_sdk/commands/format/update_readme.py

```diff
@@ -0,0 +1,135 @@
+import re
+from typing import List, Optional, Tuple
+
+import click
+
+from demisto_sdk.commands.common.constants import (RELATIVE_HREF_URL_REGEX,
+                                                   RELATIVE_MARKDOWN_URL_REGEX)
+from demisto_sdk.commands.format.format_constants import (ERROR_RETURN_CODE,
+                                                          SKIP_RETURN_CODE,
+                                                          SUCCESS_RETURN_CODE)
+from demisto_sdk.commands.format.update_generic import BaseUpdate
+
+
+class ReadmeFormat(BaseUpdate):
+    """ReadmeFormat class is designed to update integration and pack README file according to Demisto's convention.
+
+        Attributes:
+            input (str): the path to the file we are updating at the moment.
+            output_file (str): the desired file name to save the updated version to.
+    """
+
+    def __init__(self,
+                 input: str = '',
+                 output: str = '',
+                 path: str = '',
+                 no_validate: bool = False,
+                 verbose: bool = False,
+                 **kwargs):
+        super().__init__(input=input, output=output, path=path, no_validate=no_validate,
+                         verbose=verbose, **kwargs)
+        with open(self.source_file, 'r') as f:
+            self.readme_content = f.read()
+
+    def replace_url_in_content(self, relative_url: list, new_url: str):
+        """Replace the relative url link with the new url in README."""
+
+        # md link
+        if '[' in relative_url[0]:
+            old_link = relative_url[0] + '(' + relative_url[1] + ')'
+            new_link = relative_url[0] + '(' + new_url + ')'
+        # href link
+        else:
+            old_link = relative_url[0]
+            new_link = str.replace(relative_url[0], relative_url[1], new_url, 1)
+
+        self.readme_content = str.replace(self.readme_content, old_link, new_link, 1)
+        click.secho(f'Replaced {relative_url[1]} with {new_url}')
+
+    def get_relative_urls(self) -> List[list]:
```

Comment:
instead of returning a list of lists, which is making the code all around a bit confusing, lets make some objects here

`ReadMeURL(fullUrl (ifNeeded),description,link,isMarkdown)`